File: Domain/Vijest.cs
namespace Domain
{
    public class Vijest
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public string Summary { get; set; }
        public string Slug { get; set; }
        public string Author { get; set; }
        public string PictureUrl { get; set; }
        public string Category { get; set; }
        public DateTime PublishedDate { get; set; }
        public int Views { get; set; } = 0;
        public bool IsFeatured { get; set; }
        public string[] Tags { get; set; }

        // Kolekcija svih reakcija na vijest
        public ICollection<Reaction> Reactions { get; set; }
    }
}

File: client-app/src/app/models/vijest.ts
export interface Vijest {
    id?: string
    title: string
    content: string
    summary: string
    slug?: string
    author: string
    pictureUrl: string
    category: string
    publishedDate: Date | null
    views: number
    isFeatured: boolean
    tags: string[]
  }